nhais/README.md

```diff
@@ -0,0 +1,40 @@
+# NHAIS Adaptor - Developer Notes
+The following sections are intended to provide the necessary info on how to configure and run the NHAIS adaptor.
+
+## Pre-requisites
+
+Ensure you have Pipenv installed and on your path, then within nhais directory, run:
+
+`pipenv install`
+
+## Developer setup 
+
+Start IntelliJ
+
+Install Python and EnvFile plugin
+
+Restart IntelliJ
+
+Open nhais component - IntelliJ → File → Open → select nhais folder (../integration-adaptors/nhais)
+
+Open main.py file
+
+You’ll see information that there’s “No Python interpreter configured“
+
+Click “Use Pipenv interpreter“ and wait for indexing to finish
+
+Close project
```

Comment:
It is possible to define all modules without closing the project. File -> Project Structure -> Modules -> Add module

Reply:
You will be in the nhais component and not integration-adaptors at this point. I think you need to be in that root folder to add all_common to nhais.